### PR TITLE
this should address the problem with not being able to save the rmid …

### PR DESCRIPTION
**Pull request by Ito-Eta**
…change

relevant to https://www.pivotaltracker.com/story/show/185647607

### CODE REVIEW

**Comment by Ito-Eta** [comment]
@amcates ready for review and merging.

File: db/schema.rb [hunk 1]
```diff
@@ -33,6 +33,18 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
+  create_table "admin_rate_changes", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
```
Comment (amcates) [inline db/schema.rb:36]: Curious as to why this is in your pull request

Comment (Ito-Eta) [reply db/schema.rb:36]: Not sure.  I didn't touch any of that.  maybe it was in code i pulled...?  but that should have been in the main branch already, yeah?  is it causing any problems?

Comment (amcates) [reply db/schema.rb:36]: Don't think it's really an issue but I was just curious.  I will proceed with merge

**Pull request merged**